src/app/components/home/home.component.html

```diff
@@ -302,10 +303,22 @@ <h2>Change the app zoom level</h2>
       </button>
     </div>
 
+
+    <div class="horizontalDivider"></div>
+
+    <h2>{{ 'SETTINGS.changeLanguage' | translate }}</h2>
+
+    <div style="margin: 20px">
+      <select (change)="changeLanguage($event.target.value)">
```

Comment:
It might be here, but I didn't notice it:
Does this save and/or detect the language the user would like to use? 😄

Reply:
It looks like we can do something like this:
`translate.use(browserLang.match(/en|ru/) ? browserLang : 'en');`
But I'm unsure how _Electron_ handles browser language.
It might be worth adding a modal if the app detects it's the first start that has a language selection dropdown.

I'll likely add that in a different PR (I prefer smaller PRs generally: since I'm not building to _production_ every time a PR lands, I can merge in smaller features).

Reply:
Created #26 to track this!